src/javascript/app/pages/trade/tradepage.js

```diff
@@ -57,6 +58,21 @@ const TradePage = (() => {
                     if ($currency.next().attr('id') === $currency.attr('id')) $currency.next().eq(0).remove();
                 }
             });
+
+            BinarySocket.wait('website_status', 'landing_company').then(() => {
+                const is_logged_in = Client.isLoggedIn();
+                const is_eu = isEuCountry();
+                const is_malta = State.getResponse('landing_company.financial_company.shortcode') === 'maltainvest';
+
+                if ((!is_logged_in && is_eu) || (is_logged_in && is_malta)) {
+                    if ((is_logged_in && is_eu) || (is_logged_in && is_malta)) {
+                        $('#professional-cta').setVisibility(1);
+                    }
+                    $('.mfsa_message')
+                        .removeClass('container')
+                        .attr('style', 'margin-bottom: 40px'); // hello
```

Comment:
probably meant to remove this comment? 🙂

Reply:
we don't want to add the styling in javascript, it becomes hard to track. it's better to add a class that has margin-bottom: 40px defined in scss file and simply add that class here instead.